README.md

```diff
@@ -1095,6 +1095,7 @@ Graphs can be used to represent many problems in computer science, so this secti
     - [ ] [Registers and RAM (video)](https://youtu.be/fpnE6UAfbtU)
     - [ ] [The Central Processing Unit (CPU) (video)](https://youtu.be/FZGugFqdr60)
     - [ ] [Instructions and Programs (video)](https://youtu.be/zltgXvg6r3k)
+    - [ ] [Build a Modern Computer from First Principles](https://www.coursera.org/learn/build-a-computer)
```

Comment:
Let's move this down to the bottom, since it's optional information.

Reply:
already moved down: fe6eb797932e75c1ad395311bdb8d20a5a2a2df0